common/v2/components/AccountList.tsx

```diff
@@ -133,7 +127,10 @@ function buildAccountTable(
         <Network key={index} color="#a682ff">
           {account.networkId}
         </Network>,
-        <Typography key={index}>{getBalanceFromAccount(account)}</Typography>
+        <Typography key={index}>
+          {Fiats[0].symbol}
+          {total.toFixed(2)}
+        </Typography>
```

Comment:
1. Use `v2/components/currency` for the display.
2. I find it troublesome to do `Fiats[0]`. How about changing the export to an object and access it as `Fiats[USER_FIAT]` (with a better name of course).

NB. It seems that we don't need to have fiats in the localstorage. We should be able to remove it from initialisation all together. I'll ask in the slack